readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://BOHDANLIUYUI.github.io/layout_search-bar-airbnb/)
```

Comment:
[instructions](https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github) 20

![image](https://user-images.githubusercontent.com/86923089/216817858-805d9d0d-ba59-4e10-98db-fa381d7839df.png)